README.md

```diff
@@ -108,7 +108,7 @@ const shopify = new Shopify({
 
 // shopify.<resource_name>.<method_name>
 ```
-
+`shopName` can be with or without `.myshopify.com`. eg `test-shopify.myshopify.com` or `test-shopify`
```

Comment:
Can you please move this in the description of the `shopName` option? https://github.com/MONEI/Shopify-api-node#options

Also please keep the limit of 80 chars per line.
Thanks.